assets/quick-order-form.css

```diff
@@ -6,6 +6,14 @@
   background-color: rgba(var(--color-foreground), 0.02);
```

Comment:
Subtotal and total items arent updating 👀 https://screenshot.click/09-26-z1ygp-tp5ua.mp4

Reply:
I was aware of it but honestly I was sure that this would be another task. I think I saw previously a task related to the total bar. But that's okay, I can fix it here.

Reply:
Fixed here [914d27c](https://github.com/Shopify/dawn/pull/3072/commits/914d27c8f40d8b1571ef4917e44ef52a23958466).

Reply:
@sofiamatulis @andrewetchen @melissaperreault 

I'd like to clarify one thing and get your approval on it. Since the Quick Order Form is going to represent all products, the Total Bar, unlike the Quick Order List, reflects the quantity and sum of all items added to the cart. In other words, even if you add an item via the product page, it will be reflected in the Total Bar of the Quick Order Form. Please remember that we currently iterate only through one product, but the Total Bar already reflects all items in the cart. Don't be confused if you see that the data from the current form doesn't match the Total Bar.